docs/dashboard/layout.md

```diff
@@ -0,0 +1,183 @@
+
+  The size of widget and tab is set by the *Size* item.  Selecting the *Size* item will display a popup interface for specifying UI element size.  The size of the UI elemt can be specified by the grid area of this popup UI.
+
+<!--　autoボタンを押すと、widgetのサイズを自動設定します。その動作はノードによりますが、一般的に、auto設定ではwidgetの幅は所属するgroupの幅と同じ、widget毎に高さは適切な大きさが選択されます。-->
+
+  Pressing the `auto` button indicate to set the widget size automatically.  Its behaviour depends on implementation of each UI widgets.  Typically, the width of the widget is set to the same  width of the belonging group, and the height is set to appropriate value.
+
+<br style="clear:both"/>
+
+<!--　ポップアップ上部の*W x H*をクリックすると、以下の数値入力ポップアップに変化します。大きな固定数値のサイズを指定する場合はこちらが便利です。-->
+
+  Clicking *W xH* at the top of the UI will display the following interface for changing the widget size using digits input.  This UI is useful to specify the size for large widget.
+
+<img title="" src="images/layout/ElementSizerNum.png" alt="ElementSizerNum.png" data-align="inline">
+
+### Adjusting Layout using Spacer
+
+<!-- ### Spacerによるレイアウト調整 -->
+
+<img src="images/layout/Spacer.png" style="float: left; margin-right: 20px"/>
+
+<!--　Group内のwidgetの細かな配置調整のためにSpacerノードを利用できます。Spacerノードは指定したサイズの領域の穴埋めをするためのwidgetです。
+
+　GroupへのSpacerの追加は、Layoutタブのグループ上のマウスオーバで表示される+spacerボタンで行います。-->
+
+The `Spacer` node can be used for fine-tuning the placement of widgets in a group.  The Spacer node is a special widget for filling in the area with the specified size.
+
+  To add a `Spacer` to a group, use the `+spacer` button displayed while mouse over the group on the `Layout` tab.
+
+<br style="clear:both"/>
+
+### Layout Editor
+
+<img src="images/layout/LayoutButton.png" style="float:right; margin-left: 20px"/>
+
+<!--　複雑なダッシュボード画面を構成する場合、Layout Editorを用いることで、GUIによってWidgetの配置を行うことができます。Layout EditorはLayoutタブのTab上のマウス    オーバで表示される*layout*ボタンを押すことで起動します。-->
+
+  While layouting a complicated dashboard screen, GUI-based `Layout Editor` can be used.  The layout editor can be opened by pressing the `layout` button displayed while mouse over on a tab on the `Layout` tab.
+
+<br style="clear:both"/>
+
+<!--　Layout Editorでは、Tabを構成するグループのグリッドレイアウトが表示されます。
+
+　各グループの幅は右上のWidth項目によって変更できます。-->
+
+  The `Layout Editor` displays the grid layout of the groups that make up the specified tab.
+
+  The width of each group can be changed by the *Width* item displayed on the top right.
+
+<!--　グループ内のWidgetについて、drag&dropによってgroup内の配置を変更できます。Widget右上の鍵マークはWidgetのサイズがautoか否かを表します。鍵のロックが外れている場合は、右下の矢印をdragすることでサイズを変更できます。
+
+　Layout Editorでレイアウトを行うと、必要な位置にSpacerノードを自動的に挿入します。-->
+
+  The placement of widgets in a group can be changed by drag & dropping a widget.  The lock mark on the upper right of the widget indicates whether the widget size is *auto* or not.  If the key is unlocked, the widget size can be changed by dragging the arrow at the bottom right.
```

Comment:
unlocked ->   If the key is locked.... 
(it is fixed size when locked so you can set the size... if unlocked then it is auto sized)